### ISSUE TITLE
[Feature request] jb version 

### ISSUE THREAD

**Issue by karancode** [jsonnet-bundler/jsonnet-bundler#53]
Hello, 

I wanted to know is there a way to check which version of `jb` we are using. Is there a way to check it ? Something like `jb -v`.

Also, what is the way to update the `jb` version itself ? 
I have installed `jb` using 
```
go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
```

PS : New to golang (`go get`) so not sure about this.

**Comment by davidovich** [comment]
jsonnet-bundler does not have this at the moment, but you can use the go tooling to output version info:
```bash
go version -m $(which jb)
```
output:

```bash
$ go version -m $(which jb)
/home/david/go/bin/jb: go1.13
        path    github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
        mod     github.com/jsonnet-bundler/jsonnet-bundler      v0.1.1-0.20191031101518-7fc7c31856f9    h1:UHhNBlu6wu8ygmKRbMJo0OYRk2HwCbzIOtckBgcpRbs=
        dep     github.com/alecthomas/template  v0.0.0-20160405071501-a0175ee3bccc      h1:cAKDfWh5VpdgMhJosfJnn5/FoN2SRZ4p7fJNX58YPaU=
        dep     github.com/alecthomas/units     v0.0.0-20151022065526-2efee857e7cf      h1:qet1QNfXsQxTZqLG4oE62mJzwPIB8+Tee4RNCL9ulrY=
        dep     github.com/fatih/color  v1.7.0  h1:DkWD4oS2D8LGGgTQ6IvwJJXSL5Vp2ffcQg58nFV38Ys=
        dep     github.com/mattn/go-colorable   v0.0.9  h1:UVL0vNpWh04HeJXV0KLcaT7r06gOH2l4OW6ddYRUIY4=
        dep     github.com/mattn/go-isatty      v0.0.6  h1:SrwhHcpV4nWrMGdNcC2kXpMfcBVYGDuTArqyhocJgvA=
        dep     github.com/pkg/errors   v0.8.0  h1:WdK/asTD0HN+q6hsWO3/vpuAkAr+tw6aNJNDFFf0+qw=
        dep     golang.org/x/sys        v0.0.0-20190310054646-10058d7d4faa      h1:lqti/xP+yD/6zH5TqEwx2MilNIJY5Vbc6Qr8J3qyPIQ=
        dep     gopkg.in/alecthomas/kingpin.v2  v2.2.6  h1:jMFz6MfLP0/4fUyZle81rXUoxOBFi19VUFKVDOQfozc=
```

`jb -v` is implementable with the go `"runtime/debug"` package with the [`debug.ReadBuildInfo()`](https://golang.org/pkg/runtime/debug/#ReadBuildInfo) function (new in go 1.12).

> Also, what is the way to update the jb version itself ?

Outside of a go.mod enabled directory (so go get does not modify your dependencies):
```
GO111MODULE=on go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@v0.1.0
```
or

```
GO111MODULE=on go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@master
```

**Comment by metalmatze** [comment]
Very nice. We can start support the built in feature, once Go 1.14 drops and we can get rid of Go 1.11 and depend on Go 1.12+.

We could do something with build flags, like passing the commit, but that doesn't work with pure go get...

**Comment by sh0rez** [comment]
Fixed in #60
